aws-logging-monitoring/monitoring/dashboards/pod-status.json

```diff
@@ -0,0 +1,137 @@
+{
+  "displayName": "CLUSTER_NAME (Anthos cluster on AWS) pod status",
```

Comment:
> CLUSTER_NAME

I assume we're going to have instructions that will cause this to be replaced. No action required.

Aside: Once they exist, maybe we should add a README.md file somewhere within aws-logging-monitoring that links back to our docs?

Reply:
Yes `CLUSTER_NAME` will be replaced in the usage instruction, which is referenced in `aws-logging-monitoring/monitoring/README.md`.